### PR TITLE
[#6812] blocks: add self hosted video block 

### PR DESCRIPTION
**Pull request by phillimorland**

### CODE REVIEW

**Review by khamui — COMMENTED** [review]
cool! the short video description has a different font than for instance the caption of an image. this is intended?

and another comment and a question :)

File: apps/projects/models.py [hunk 1]
```diff
@@ -27,7 +26,6 @@
     ('paragraph', blocks.RichTextBlock(icon="pilcrow")),
     ('image', ImageChooserBlock(icon="image")),
     ('aligned_image', core_blocks.AlignedImageBlock(icon="image")),
-    ('video', EmbedBlock(icon="media")),
```
[the remark this answers is not in the document]
Comment (phillimorland) [reply apps/projects/models.py:30]: it's not legally compliant so I hope not?! Kathi said to take it out.

File: apps/core/templatetags/core_tags.py [hunk 2]
```diff
@@ -40,3 +40,17 @@ def combined_url_parameter(request_query_dict, **kwargs):
         combined_query_dict.setlist(key, [kwargs[key]])
     encoded_parameter = '?' + combined_query_dict.urlencode()
     return encoded_parameter
+
+
+@register.simple_tag
+def file_type(media_file):
+    if media_file.endswith('.mp4'):
+        return 'video/mp4'
+    if media_file.endswith('.webm'):
```
[the remark this answers is not in the document]
Comment (phillimorland) [reply apps/core/templatetags/core_tags.py:49]: > cool! the short video description has a different font than for instance the caption of an image. this is intended?
> 
> and another comment and a question :)

no it should look the same, will check it out!

**Comment by phillimorland** [comment]
@khamui while checking the formatting I realised that the image block was not using template so add a fix so more then initial PR

**Comment by khamui** [comment]
very nice!

**Pull request merged**